src/openzaak/components/documenten/api/viewsets.py

```diff
@@ -431,6 +443,136 @@ def _zoek(self, request, *args, **kwargs):
+            "Het uploaden van een metadata bestand, ter gebruik voor de IMPORT. "
+            "Deze actie is niet beschikbaar wanneer de `CMIS_ENABLED` optie is "
+            "ingeschakeld. Deze actie start tevens de IMPORT. Één actieve IMPORT "
+            " tegelijkertijd is mogelijk. De volgende kolommen worden verwacht "
+            f"(op volgorde) in het bestand: {_get_import_headers()}. Voor deze "
+            "actie is een APPLICATIE nodig met `heeft_alle_autorisaties` ingeschakeld."
+        ),
+        request={"text/csv": OpenApiTypes.BYTE},
+        responses={
+        status.HTTP_200_OK: OpenApiTypes.NONE,
+        **VALIDATION_ERROR_RESPONSES,
+        **COMMON_ERROR_RESPONSES,
+        },
+    )
+)
+class EnkelvoudigInformatieObjectImportUploadView(ImportUploadView):
+    import_type = ImportTypeChoices.documents
+    import_headers = DocumentRow.import_headers
+
+    required_scopes = {}
+
+    @property
+    def import_dir(self) -> Path:
+        return Path(settings.IMPORT_DOCUMENTEN_BASE_DIR)
+
+    def create(self, request, *args, **kwargs):
+        if settings.CMIS_ENABLED:
+            raise CMISNotSupportedException()
+
+        response = super().create(request, *args, **kwargs)
+
+        import_instance = self.get_object()
+        import_documents.delay(import_instance.pk)
+
+        return response
+
+
+@extend_schema_view(
+    retrieve=extend_schema(
+        operation_id="enkelvoudiginformatieobject_import_status",
+        summary=_("De status van een IMPORT opvragen."),
+        description=mark_experimental(
+            "Het opvragen van de status van een IMPORT. Deze actie is niet "
+            "beschikbaar wanneer de `CMIS_ENABLED` optie is ingeschakeld. "
+            "Voor deze actie is een APPLICATIE nodig met `heeft_alle_autorisaties`"
+            " ingeschakeld."
+        ),
+    )
+)
+class EnkelvoudigInformatieObjectImportStatusView(ImportStatusView):
+    import_type = ImportTypeChoices.documents
+
+    required_scopes = {}
+
+    def retrieve(self, request, *args, **kwargs):
+        if settings.CMIS_ENABLED:
+            raise CMISNotSupportedException()
+        return super().retrieve(request, *args, **kwargs)
+
+
+def _get_report_headers():
+    return ", ".join([f"**{header}**" for header in DocumentRow.export_headers])
+
+
+@extend_schema_view(
+    retrieve=extend_schema(
+        operation_id="enkelvoudiginformatieobject_import_report",
+        summary=_("Het reportage bestand van een IMPORT downloaden."),
+        description=mark_experimental(
+            "Het reportage bestand downloaden van een IMPORT. Dit bestand is alleen "
+            "beschikbaar indien de IMPORT is afgerond (ongeacht het resultaat). "
+            "Deze actie is niet beschikbaar wanneer de `CMIS_ENABLED` optie is "
+            "ingeschakeld. De volgende kolommen zijn te vinden in het rapportage "
+            f"bestand: {_get_report_headers()}. Voor deze actie is een APPLICATIE"
```

Comment:
Currently there is no space between `APPLICATIE` and `nodig`
```suggestion
            f"bestand: {_get_report_headers()}. Voor deze actie is een APPLICATIE "
```